### DISCUSSION TITLE
Is there any reason for "grounded alias" to "Grounded" transition not being "inertialization" for "blend logic" in AB_Als_Locomotion?

### DISCUSSION THREAD

**Discussion by ameaninglessname** [Sixze/ALS-Refactored#237] [Q&A] [answered]
I want to set it to "inertialization" to solve some blending problem from "climbing" to "grounded" but worrying about break potential design.

**Comment by Sixze** [comment, marked as answer]
I'm not exactly sure about the reason, the original plugin does not use inertialization either, I think there is simply no need for it.
For your case, I would just add separate transitions from new states to "Grounded", so that you don't have to change existing transitions.
